src/common/cluster-store/allowed-resources.injectable.ts

```diff
@@ -4,15 +4,15 @@
  */
 import { getInjectable } from "@ogre-tools/injectable";
 import { comparer, computed } from "mobx";
-import hostedClusterInjectable from "./hosted-cluster.injectable";
+import hostedClusterInjectable from "../../renderer/cluster/hosted-cluster.injectable";
```

Comment:
This import is not welcome. Files in `common` should never import from `renderer` or `main` directly. `common` injectables are available in both processes, and it would cause instantiation to throw exception. Separate ongoing discussion in slack to figure out how to solve it.

Reply:
I don't see the reason why we need to tackle this in this PR. Maybe you could just revert the move on `hostedCluster` even though we know it will be in wrong place.

Reply:
Ok I will revert this.